recipes/geocat-comp/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "geocat-comp" %}
+{% set version = "2022.03.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/{{ name }}/archive/v{{ version }}.tar.gz   # For building from a specific tag's tarball
+  sha256: ec73166e12f20867821365ef8467d970288132093f7a680c206ab2d661d61c4d
+  # git_tag: v{{ version }}   # For building from a specific tag
+  # git_rev: main # For building from main branch for debugging purposes
+  # git_url: https://github.com/NCAR/{{ name }}.git
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - python                              # [build_platform != target_platform]
+    - pip                                 # [build_platform != target_platform]
+    - setuptools                          # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - cf_xarray >=0.3.1
+    - cftime
+    - dask
+    - eofs
+    - geocat-f2py
```

Comment:
https://github.com/conda-forge/geocat-f2py-feedstock/issues/4

It doesn't seem that geocat-f2py can be made into noarch, or am I mistaken? Your pypi wheels are noarch tho...

Reply:
geocat-f2py doesn't actually work if installed via pypi, I was not part of that, since I was on leave at the time, but I have been assigned to look into it after we have stable distribution through conda-forge. 

As it stands geocat-f2py's compilation is more complex than it looks and is most definitely not a noarch package.

Reply:
Okay, I left a pointer in the issue: https://github.com/conda-forge/climlab-feedstock

It could be as easy as adding the right deps and making a .bat script, but I am not windows fluent 😆

Reply:
thanks.